tests/phpunit/AdapterFactoryTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\FtpExtractor\Tests;
+
+use Keboola\FtpExtractor\AdapterFactory;
+use Keboola\FtpExtractor\Config;
+use Keboola\FtpExtractor\ConfigDefinition;
+use League\Flysystem\Adapter\Ftp;
+use League\Flysystem\Sftp\SftpAdapter;
+use PHPUnit\Framework\TestCase;
+
+class AdapterFactoryTest extends TestCase
+{
+    public function testGetFtpAdapter(): void
```

Comment:
JFYI dalo by se klidně napsat jako @dataProvider

Reply:
https://phpunit.de/manual/6.5/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers

Reply:
jo, tady se hodí. Vypadá to přehledněji.